src/bptl/tasks/tests/test_add_defaultservices.py

```diff
@@ -0,0 +1,142 @@
+from unittest.mock import patch
+
+from django.test import TestCase, override_settings
+from django.urls import reverse_lazy
+
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+from bptl.accounts.models import User
+from bptl.work_units.zgw.models import DefaultService
+
+from ..forms import DefaultServiceFormset
+from ..models import TaskMapping
+from ..registry import WorkUnitRegistry
+
+# Set up an isolated registry for tests
+test_register = WorkUnitRegistry()
+
+
+@test_register
+def task1(task):
+    """Task 1 documentation"""
+    pass
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        "bptl.accounts.backends.UserModelEmailBackend",
+        "django.contrib.auth.backends.ModelBackend",
+    ]
+)
+class AddDefaultServicesTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = User.objects.create_superuser("super", "user@utrecht.nl", "letmein")
+
+    def setUp(self):
+        self.client.login(username="super", password="letmein")
+
+        patcher = patch("bptl.tasks.forms.register", new=test_register)
+        self.mocked_register = patcher.start()
+        self.addCleanup(patcher.stop)
+
+        self.url = reverse_lazy("tasks:taskmapping-create")
+
+        self.data = {
```

Comment:
For the future: django-webtest makes your life so much easier when dealing with forms/formsets :)

Reply:
I know, but when I tried to use `django-webtest` in response.form the `service` fields were missing (`alias` fields were there though). In browser everything was ok, so I just switched to django testcases